clusterctl/clusterdeployer/clusterapiserver.go

```diff
@@ -73,7 +73,8 @@ func getApiServerCerts() (*caCertParams, error) {
 	return certParams, nil
 }
 
-func getApiServerYaml() (string, error) {
+// GetApiServerYaml returns the clusterapi-apiserver manifest used for deployment
+func GetApiServerYaml() (string, error) {
```

Comment:
If we are going to expect tooling outside of clusterctl to use this, should we move it under the `/pkg` directory instead? I would assume that anything under `/clusterctl` would be private to clusterctl (or leveraged by provider-specific clusterctl builds until the extension mechanisms are fully fleshed out).

Reply:
@detiber Thanks, I think moving clusterapiserver.go to /pkg makes sense!
I could use `/pkg/util` , or we could have a new `/pkg/deploy`. Do you have a preference for the pkg name?

Reply:
I'll make this `pkg/deployer` if no objections

Reply:
sounds good to me